docs/faq.md

```diff
@@ -756,3 +771,54 @@ On examining the container structure with [Dive](https://github.com/wagoodman/di
 The user had used Jib's ability to install extra files into the image ([Maven](https://github.com/GoogleContainerTools/jib/tree/master/jib-maven-plugin#adding-arbitrary-files-to-the-image), [Gradle](https://github.com/GoogleContainerTools/jib/tree/master/jib-gradle-plugin#adding-arbitrary-files-to-the-image)) to install a library file by placing it in `src/main/jib/lib/libfoo.so`. This would normally cause the `libfoo.so` to be installed in the image as `/lib/libfoo.so`. But `/lib` and `/lib64` in the user's base image were symbolic links. Jib does not follow such symbolic links when creating the image. And at container initialization time, Docker treats these symlinks as a file, and thus the symbolic link was replaced with `/lib` as a new directory. As a result, none of the system shared libraries were resolved and dynamically-linked programs failed.
 
 Solution: The user installed the file in a different location.
+
+## Jib CLI 
+ 
+### How does the `jar` command support Standard JARs?
+
+A standard JAR can be containerized by the `jar` command in two modes, exploded or packaged. 
+
+#### Exploded Mode (Recommended)
+Achieved by calling `jib jar --target ${TARGET_REGISTRY} ${JAR_NAME}.jar`
+
+The default mode for containerizing a JAR. It will open up the JAR into the following layers:  
+
+- Other Dependencies Layer
+- Snapshot-Dependencies Layer
+- Resources Layer
+- Classes Layer
+
+**Entrypoint** : `java -cp /app/dependencies/:/app/explodedJar/ ${MAIN_CLASS}`
+
+#### Packaged Mode
+Achieved by calling `jib jar --target ${TARGET_REGISTRY} ${JAR_NAME}.jar --mode packaged`.
+
+It will result in the following layers on the container:
+- Dependencies Layer
+- Jar Layer
+
+**Entrypoint** : `java -jar ${JAR_NAME}.jar`
+
+### How does the `jar` command support Spring Boot JARs?
+The `jar` command currently supports containerization of Spring Boot Fat JARs.
+A Spring-Boot Fat JAR can be containerized in two modes, exploded or packaged. 
+
+#### Exploded Mode (Recommended)
+Achieved by calling `jib jar --target ${TARGET_REGISTRY} ${JAR_NAME}.jar`
+
+The default mode for containerizing a JAR. It will respect `layers.idx` (if present) or create layers in the following format:
+
+- Other Dependencies Layer
+- Spring-Boot-Loader Layer
+- Snapshot-Dependencies Layer
+- Resources Layer
+- Classes Layer
```

Comment:
For consistency with the [other FAQ](https://github.com/GoogleContainerTools/jib/blob/02c9b5ce3e40c80f93ee900e4903c6d2cdc71cee/docs/faq.md#how-are-jib-applications-layered), how about reversing the layer order?

Reply:
BTW, is the order correct? Why is the spring boot loader in between the two dep layers?

Reply:
Ah this is how we ordered the layers for the non-layered JAR - the purpose was to make it similar to the default order defined by Spring Boot (when `layers.idx` is specified) https://spring.io/blog/2020/08/14/creating-efficient-docker-images-with-spring-boot-2-3

Reply:
Hm for reversing the ordering: this is how the layers are currently ordered in the cli so I wonder if it is better to leave it as it is to represent that?

Reply:
Well then just reverse the order in the other FAQ. In either way, we can achieve consistency.